backend/index.js

```diff
@@ -109,11 +109,103 @@ app.get('/patchnotes/valorant/:version', async (req, res) => {
     try {
         const patchDetails = await prisma.patchnotes_val.findMany({
             where: { text: `valorant-patch-notes-${version}` },
-            include: { Agents: true, Maps: true, Bugs: true },
+            include: { Agents: true, Maps: true, Bugs: true, comments: { include: { user: true }, }, },
         });
         res.json(patchDetails);
     } catch (error) {
         console.error('Error in /patchnotes/valorant/:version route:', error.message);
         res.status(500).send({ error: 'Error while fetching patch notes details' });
     }
 });
+
+app.post('/patchnotes/overwatch/:year/:month/comments', async (req, res) => {
+    const { message, patchId, userId } = req.body;
+    try {
+        const newComment = await prisma.comment_ow.create({
+            data: {
+                message,
+                patchId,
+                userId,
+            },
+        });
+        res.json(newComment);
+    } catch (error) {
+        console.error(error);
+        res.status(500).json({ error: 'Failed to create comment' });
+    }
+});
+
+app.post('/patchnotes/valorant/:version/comments', async (req, res) => {
```

Comment:
Do you plan on splitting out each of these fetches to only happen on the relevant pages ultimately? Seems like we wouldn't ultimately want to do all of this within `index.js`

Reply:
I planned on splitting them up depending on the game. Overwatch and Valorant would have their own files for rendering the comments, votes, replies, etc.